### ISSUE TITLE
🚀 Feature Request: Stay in column should be enabled by default

### ISSUE THREAD

**Issue by Michaelgimii** [elementor/elementor#19301]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

In elementor, we can enable the option to [stay in the column](https://prnt.sc/URQEMeZlugcu). But one question. Why should this option exist?
In fact, isn't it better to have this option enabled by default for containers and sections? I can't imagine any situation where users wouldn't need this option. Therefore, I think it is better to remove this option and make it active by default. Users do not need to choose whether this option is active or not. This is what we need anyway. If we want to make a section or container sticky.

### Describe the solution you'd like

remove the stay in column option and enable it by default

**Comment by perks98** [comment]
I completely agree that if a section is set to sticky that staying in the column should be enabled by default but where my thought differs is that I can still see value in having an area lower on the page where it becomes fixed once you have scrolled to that section and for backward compatibility purposes. I have seen other designers implement this by having a page-specific sticky navigation banner within the page content that link to anchor points on the same page.

If you look at the section titled 'Built For Every Industry' on desktop view you can see Elementor using this: https://elementor.com/for/agency/

**Comment by 79ho3ein** [comment]
+1
